.ddev/config.yaml

```diff
@@ -0,0 +1,33 @@
+name: pixelfed
```

Comment:
```suggestion
```

Recommend leaving this out, so it will just take the directory name.

Reply:
Done